components/ai_chat/browser/ai_chat_tab_helper.h

```diff
@@ -115,7 +115,19 @@ class AIChatTabHelper : public content::WebContentsObserver,
                         const GURL& icon_url,
                         bool icon_url_changed,
                         const gfx::Image& image) override;
-
+  std::string BuildClaudePrompt(const std::string& question_part,
```

Comment:
nit: might be worth moving this prompt building functions to a `_util` file to separate from overarching logic

Reply:
That's described at https://github.com/brave/brave-browser/issues/31623